### PR TITLE
all: update to netty 4.1.5

### LINKED ISSUE #2224: Update to netty 4.1.5.final
Long story short, there was a bug until 4.1.5.final that would cause issues when running inside Tomcat.

### PR DESCRIPTION
**Pull request by carl-mastrangelo**
Fixes #2224

### CODE REVIEW

**Comment by carl-mastrangelo** [comment]
The Netty bug has been fixed, PTAL

**Comment by carl-mastrangelo** [comment]
@zhangkun83 PTAL?

File: build.gradle [hunk 1]
```diff
@@ -158,9 +158,9 @@ subprojects {
                 protobuf_plugin: 'com.google.protobuf:protobuf-gradle-plugin:0.8.0',
                 protobuf_util: "com.google.protobuf:protobuf-java-util:${protobufVersion}",
 
-                netty: 'io.netty:netty-codec-http2:[4.1.4.Final]',
-                netty_epoll: 'io.netty:netty-transport-native-epoll:4.1.4.Final' + epoll_suffix,
-                netty_tcnative: 'io.netty:netty-tcnative-boringssl-static:1.1.33.Fork19',
+                netty: 'io.netty:netty-codec-http2:[4.1.5.Final]',
+                netty_epoll: 'io.netty:netty-transport-native-epoll:4.1.5.Final' + epoll_suffix,
+                netty_tcnative: 'io.netty:netty-tcnative-boringssl-static:1.1.33.Fork22',
```
Comment (ejona86) [inline build.gradle]: Also update SECURITY.md?

**Comment by ejona86** [comment]
Do you know what the Windows failure is about? It seems likely legit.

**Comment by carl-mastrangelo** [comment]
@ejona86 I do not, but filed https://github.com/netty/netty-tcnative/issues/188

**Comment by carl-mastrangelo** [comment]
I'm going to rollback this change

**Pull request closed without merging**